Gigs/Gigs/View Controllers/GigsTableViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  GigsTableViewController.swift
+//  Gigs
+//
+//  Created by Jonathan Ferrer on 5/16/19.
+//  Copyright © 2019 Jonathan Ferrer. All rights reserved.
+//
+
+import UIKit
+
+class GigsTableViewController: UITableViewController {
+
+    let gigController = GigController()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
```

Comment:
Wrong place, all this code need to happen on viewDidAppear so that it gets executed when you dismiss the view when you log in.

```
 override func viewDidAppear(_ animated: Bool) {
        super.viewDidAppear(true)
        
        if gigController.bearer == nil {
            performSegue(withIdentifier: "LoginViewModalSegue", sender: self)
        } else {
            gigController.getAllGigs { (result) in
                
                do{
                    self.gigController.gigs = try result.get()
                    
                    DispatchQueue.main.async {
                        self.tableView.reloadData()
                    }
                } catch {
                    NSLog("Errror getting all gigs")
                }
            }
        }
    }
```